lib/ansible/modules/network/netact/netact_cm_command.py

```diff
@@ -0,0 +1,364 @@
+    To set the scope of the operation, define Distinguished Name (DN) or Working Set (WS) or
+    Maintenance Region (MR) as input.
+options:
+    operation:
+        description:
+            operation
+            Supported operations are upload, provision, import, export and Provision_Mass_Modification.
+        required: true
+        choices:
+            - upload
+            - provision
+            - import
+            - export
+            - Provision_Mass_Modification
+        aliases:
+            - op
+    opsName:
+        description:
+            - user specified operation name
+        required: false
+    DN:
+        description:
+            Sets the exact scope of the operation in form of a list of managed object Distinguished Names (DN)
+             in the network.
+            A single DN or a list of DNs can be given (comma separated list without spaces).
+            Alternatively, if DN or a list of DNs is not given, working set (WS) or Maintenance Region (MR)
+            must be provided as parameter to set the scope of operation.
+        required: false
+
+    WS:
+        description:
+            Sets the scope of the operation to use one or more pre-defined working sets (WS) in NetAct.
+            A working set contains network elements selected by user according to defined criteria.
+            A single WS name, or multiple WSs can be provided (comma-separated list without spaces).
+            Alternatively, if a WS name or a list of WSs is not given, Distinguished Name (DN) or
+            Maintenance Region(MR) must be provided as parameter to set the scope of operation.
+        required: false
+    MR:
+        description:
+            Sets the scope of the operation to network elements assigned to a Maintenance Region (MR)
+            Value can be set as MR IDs including the Maintenance Region Collection (MRC)
+            information (for example MRC-FIN1/MR-Hel).
+            Multiple MRs can be given (comma-separated list without spaces)
+            The value of this parameter is searched through MR IDs under given MRC. If there is no match,
+            then it is searched from all MR names.
+            Alternatively, if MR ID or a list or MR IDs is not given, Distinguished Name (DN) or Working Set (WS)
+            must be provided as parameter to set the scope of operation.
+        required: false
+    planName:
+        description:
+            - Specifies a plan name.
+        required: false
+    type:
+        description:
+             Specifies the type of the export operation.
+        required: true
+        choices:
+          - plan
+          - actual
+          - reference
+          - template
+          - siteTemplate
+    fileFormat:
+        description:
+            Indicates file format.
+        required: false
+        choices:
+            - RAML2
+            - CSV
+            - XLSX
+    fileName:
+        description:
+            - Specifies a file name.
```

Comment:
If you update the argspec as I suggested you can change this to something like:
`Exactly one of I(inputFile) and I(fileName) is required.`